docs/cudf/source/developer_guide/developer_guide.md

```diff
@@ -0,0 +1,129 @@
+# Developer Guide
+
+```{note}
+At present, this guide only covers the main cuDF library.
+In the future, it may be expanded to also cover dask_cudf, cudf_kafka, and custreamz.
+```
+
+The cuDF library is a GPU-accelerated, [Pandas-like](https://pandas.pydata.org/) DataFrame library.
```

Comment:
Later on you just use `cuDF` rather than "The cuDF library". Maybe start with that here?
```suggestion
cuDF is a GPU-accelerated, [pandas-like](https://pandas.pydata.org/) DataFrame library.
```

(pandas is I think canonically lowercase)